co2eq/flights/index.js

```diff
@@ -1,12 +1,54 @@
 import { geoDistance } from 'd3-geo'; // todo - add d3-geo in package.json
+import {
+  ACTIVITY_TYPE_TRANSPORTATION,
+  TRANSPORTATION_MODE_PLANE,
+} from '../../definitions';
+
 import airports from './airports.json';
 import { getActivityDurationHours } from '../utils';
+import loadfactors from './loadfactors.json'
+
+// Constants for JSON keys
+const ICAO_REGION_KEY = 'icao_region_code';
+const PASSENGER_LOAD_FACTORS_KEY = 'passenger_load_factors';
+const PASSENGER_FREIGHT_RATIO_KEY = 'passenger_to_freight_ratio';
+
+
+export const modelName = 'flight';
+export const modelVersion = '1';
+export const explanation = {
+    text: 'Calculations take into account direct emissions from burning fuel and manufacturing of vehicle. They are based on international statistics on passenger and cargo loads and aircraft type usage.',
+    links: [
+      { label: 'My Climate (2019)', href: 'https://www.myclimate.org/fileadmin/user_upload/myclimate_-_home/01_Information/01_About_myclimate/09_Calculation_principles/Documents/myclimate-flight-calculator-documentation_EN.pdf'},
+      { label: 'ICCT Report (2018)', href: 'https://theicct.org/sites/default/files/publications/ICCT_CO2-commercl-aviation-2018_20190918.pdf'},
+    ],
+}
+
+export const modelCanRunVersion = 1;
+export function modelCanRun(activity) {
+  const {
+    activityType,
+    transportationMode,
+    distanceKilometers,
+    durationHours,
+    departureAirportCode,
+    destinationAirportCode,
+  } = activity;
+  if ((activityType === ACTIVITY_TYPE_TRANSPORTATION)
+    && (transportationMode === TRANSPORTATION_MODE_PLANE)
+    && (distanceKilometers || durationHours || (departureAirportCode && destinationAirportCode))
```

Comment:
durationHours doesn't exist anymore. Maybe you meant endDatetime?

Reply:
I updated that 😅 see in the last commits?

Reply:
hmm. This is the last version, so something must have gone wrong?

Reply:
Yes something went wrong after I fetched upstream and then rebased , see here for code with updated `endDateTime` https://github.com/tmrowco/northapp-contrib/tree/17f9a9b7849d3f79455cc9477e081f77ed4f2019/co2eq
It seems like the branch was duplicated or something like that, I'll look into it tomorrow :)

Reply:
Oh I didn't realize that it appeared here, I'll update that + I finally managed to rebase and squash everything together. But I don't know why the README.md update and trainline update appear in the PR ...

Reply:
hum I think if you `git fetch origin master:master` and `git merge master` then the branch will be up to date (here it says you are behind master)

Reply:
In the commit history from the branch feature/load_factors is see this:

```
pick 7141633 uses moment for safely parsing datetimes in utils
pick 9013460 fix trainline
pick 68b138e fix bug
pick fa8f1b6 Updated co2model description.2
pick bfbf759 Added ICAO region codes to airports.json.
pick a7924cb Add loadfactors per ICAO regions.
pick a00d340 Variable load factors taken into consideration.
pick 0892be1 Simple changes requested in review.
pick 8ed2998 Resolve no-shadow lint.
pick 18c4749 Proposal for new plane model.
pick 1f65d19 Update versions.
```

So I don't understand why it says I am behind master. I can't run the fetch command (on both master and feature/load_factors it returns `fatal: Refusing to fetch into current branch refs/heads/master of non-bare repository`. I have never seen that so I am a bit clueless. But I think that it wouldn't create any problem as the weird added files are already merged on master